hudi-client/src/test/java/org/apache/hudi/TestUpdateSchemaEvolution.java

```diff
@@ -16,10 +16,9 @@
  * limitations under the License.
  */
 
-package org.apache.hudi.func;
+package org.apache.hudi;
```

Comment:
IMO, `org.apache.hudi` is a very common( or say generic) level. It would be better to make the specific classes host in a detailed package that can represent his location. WDYT?

Reply:
+1

Reply:
I will move classes out to specific packages.. The test files I left out there are kind of cross functional across packages.. But agree. let me make one more pass